internal/interfaces/http/tdex_connect_handler.go

```diff
@@ -133,7 +146,7 @@ func (t *tdexConnect) AuthHandler(w http.ResponseWriter, req *http.Request) {
 
 	// append the macaroon
 	connectUrl, err = tdexdconnect.EncodeToString(
-		t.serverPort, cert, macaroon,
+		t.scheme, t.serverAddress, cert, macaroon,
```

Comment:
shoudnt here user the config OPERATOR_URL_HOST?

Reply:
If I pass then `TDEX_CONNECT_URL_HOST=127.0.0.1:9000` it will badly concatednate server addr with the host

<img width="1442" alt="image" src="https://user-images.githubusercontent.com/3596602/186645214-f7b5c5c2-b3be-4afe-9203-cb3205fe59f1.png">

Reply:
in ref to [this comment](https://github.com/tdex-network/tdex-daemon/pull/625#discussion_r954812264), here we should override the host of server address in case `CONNECT_URL_HOST` is defined.

Reply:
ok then we must pass `CONNECT_URL_HOST` to this method